cpp/src/parquet/parquet_version.h.in

```diff
@@ -18,14 +18,16 @@
 #ifndef PARQUET_VERSION_H
 #define PARQUET_VERSION_H
 
-#define PARQUET_VERSION_MAJOR @PARQUET_VERSION_MAJOR@
-#define PARQUET_VERSION_MINOR @PARQUET_VERSION_MINOR@
-#define PARQUET_VERSION_PATCH @PARQUET_VERSION_PATCH@
+#include "arrow/util/config.h"
```

Comment:
We don't need this because `@ARROW_*@` are expanded in build phase.

Reply:
removed.